BrowserKit/Package.resolved

```diff
@@ -41,8 +41,17 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/getsentry/sentry-cocoa.git",
       "state" : {
-        "revision" : "5575af93efb776414f243e93d6af9f6258dc539a",
-        "version" : "8.36.0"
+        "revision" : "7fc7ca43967e2980d8691a8e017c118a84133aac",
+        "version" : "8.26.0"
+      }
+    },
+    {
+      "identity" : "swiftdraw",
+      "kind" : "remoteSourceControl",
+      "location" : "https://github.com/swhitty/SwiftDraw",
+      "state" : {
+        "revision" : "a5c680f07b33f4cc9a451491b417b8119de23c6e",
+        "version" : "0.17.0"
```

Comment:
lets keep the same resolved file and remove these changes

Reply:
@cyndichin  removed BrowserKit/Package.resolved file from PR

Reply:
it seems the file was deleted, could we remove any changes from being committed related to that file instead? Basically, avoid having it show up in the commit changes.

Reply:
@cyndichin reverted to initial Package.resolved(hopefully).